microhapulator/data/template.html

```diff
@@ -241,7 +241,7 @@ <h3>Interpretation Hints</h3>
                             <th>Marker</th>
                             {% for i in range(len(mapping_rates)) %}
                             <th class="alnrt" style="font-weight:bold" data-sortas="numeric">Reads</th>
-                            <th class="alnrt" style="font-weight:bold" data-sortas="numeric">Off Target</th>
+                            <th class="alnrt" style="font-weight:bold" data-sortas="numeric">Repetitive Off Target</th>
```

Comment:
The donut plots use "off target" to mean reads that don't align to any marker sequences, so I added "repetitive" here to clarify these reads map to marker sequences but preferentially align elsewhere in the genome. It still might be a bit confusing so I'm open to other ideas to distinguish these cases.

Reply:
The interpretation help description is pretty clear in describing the four categories of reads, so I think we just need to use a concise and distinct label for each category. As we discussed briefly in person, I think "repetitive off target" could be changed to just "repetitive" so as not to create any confusion with the "off target" category.